static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -45,12 +45,12 @@ define([
         }),
 
         AU: assign({}, membershipParams, {
-            messageText: 'We need you to help support our fearless independent journalism. Become a Guardian Australia member for just $10 a month.',
+            messageText: 'We need you to help support our fearless independent journalism. Become a Guardian Australia member for just $1.92 a week.',
             campaignCode: "mem_au_banner"
         }),
 
         INT: assign({}, membershipParams, {
-            messageText: 'For less than the price of a coffee a week, you could help secure the Guardian\'s future. Support our journalism for $7 / €5 a month.',
+            messageText: 'For less than the price of a coffee a week, you could help secure the Guardian\'s future. Support our journalism for $1.33 / €0.95 a week.',
```

Comment:
At least in Germany you would write it rather like `0.95 €`. I'm not sure about other countries, but maybe it's worth double checking :)

Edit: Wikipedia has a nice overview: https://en.wikipedia.org/wiki/Linguistic_issues_concerning_the_euro in the Summary table.